azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# ASP.NET
```

Comment:
@tpaulshippy, not sure if you knw but there are already pipeline files for build/test.  If you would like to add a step to publish the artifacts, the existing file can be found [here](https://github.com/Azure-Samples/eventgrid-viewer-blazor/blob/main/infrastructure/azure-pipelines/templates/steps/build-test-steps-template.yml).  If you decide to do this, please change the file to *build-test-publish-steps-template.yml*

I had planned on doing the following (feel free to do that as part of the PR):
- change trigger to main
- rename the yml files and replace master with main